magento/ece-tools
# MCLOUD-6120: Crypt key absence message appear on installation process

### Description

During installation of Magento `crypt key` is generated automatically.
Now `cloud.log` contains notice that this key is absent even when Magento was installed:
```    
    [2020-06-02 14:10:08] NOTICE: Crypt key missing. Add the crypt key to the "app/etc/env.php" file, or set the "CRYPT_KEY" environment variable in the Cloud Project web UI.  
    [2020-06-02 14:10:08] INFO: Checking if db exists and has tables  
    [2020-06-02 14:10:08] NOTICE: Starting install.  
    [2020-06-02 14:10:08] INFO: Installing Magento. 
```

Need to remove that message even for upgrade process as it duplicates after implementation https://github.com/magento/ece-tools/pull/732

### Fixed Issues (if relevant)

1. https://jira.corp.magento.com/browse/MCLOUD-6120

### Manual testing scenarios

1. Uninstall Magento and run redeployment
**Expected result:** magento was installed. cloud.log does not have notice that crypt key does not exist
2. on installed Magento remove crypt/key from app/etc/env.php file and run redeployment
**Expected result:** deployment was failed with error that crypt key does not exist.
3. Add CRYPT_KEY cloud environment variable (and run redeployment)
**Expected result:** Magento upgraded successfully
4. Uninstall Magento and run redeployment (CRYPT_KEY exists from prev. step)
**Expected result:** magento was installed. env.php file has `crypt/key` the same as in `CRYPT_KEY` variable
5. Remove CRYPT_KEY cloud environment variable (and run redeployment)
**Expected result:** Magento upgraded successfully

### Contribution checklist
